project/Common.scala

```diff
@@ -88,5 +87,5 @@ object Common extends AutoPlugin {
     Test / parallelExecution := false,
     publishTo := sonatypePublishTo.value,
     sonatypeProfileName := "com.lightbend",
-    gpgKey := Some("4704549B8310E30C64805EFB7A4A132FB335FFFE"))
+    usePgpKeyHex("4704549B8310E30C64805EFB7A4A132FB335FFFE"))
```

Comment:
this key was created and encrypted  previously, trying that one before creating a new one